### ISSUE TITLE
AttributeError: 'INSwapper' object has no attribute 'taskname'

### ISSUE THREAD

**Issue by Saganaki22** [kijai/ComfyUI-LivePortraitKJ#84]
I have correct insightface, onnx, onnxruntime-gpu version for my python + portable comfyUI version
I have buffalo_l and insightface onnx models

I did all the Reactor node troubleshooting

Win10
Portable ComfyUI
NVIDIA 3090

still same issue

Anyone got any clues

![Screenshot 2024-07-24 131931](https://github.com/user-attachments/assets/cf46681e-cc6f-42d1-8cce-88b64d1c5b40)

**Comment by melMass** [comment]
Can you show the layout of insightface folder? (At a few level if it makes sense) from what I could find online it's either an issue with your onnx version or the layout of the buffalo directory

**Comment by Saganaki22** [comment]
I have reinstalled both onnxruntime and onnxruntime-gpu to the correct versions outlined in the reactor node troubleshooting

# Folder Structure

insightface
├── models
│   └── buffalo_l
│       ├── 1k3d68.onnx
│       ├── 2d106det.onnx
│       ├── det_10g.onnx
│       ├── genderage.onnx
│       ├── inswapper_128.onnx
│       └── w600k_r50.onnx

**Comment by melMass** [comment]
Those are exclusive package(regular and gpu) but it still shouldn't give this kind of error

**Comment by Saganaki22** [comment]
> Those are exclusive package(regular and gpu) but it still shouldn't give this kind of error

what i find odd is that i use insightface in other workflows and i have not had issues like this

**Comment by melMass** [comment]
That's indeed weird, I just checked and the layout seems fine.
Searching your error gives a lot of hits on the Reactor issue tracker.

And I've seen various errors that can happen with the non gpu version of onnnx but it's not that.

If you are able to run `pip freeze` in your comfy python environment it might help spot dependency issues.

Can you confirm that the other workflow were you used insightface still works now too tbs?

**Comment by Saganaki22** [comment]
> That's indeed weird, I just checked and the layout seems fine. Searching your error gives a lot of hits on the Reactor issue tracker.
> 
> And I've seen various errors that can happen with the non gpu version of onnnx but it's not that.
> 
> If you are able to run `pip freeze` in your comfy python environment it might help spot dependency issues.
> 
> Can you confirm that the other workflow were you used insightface still works now too tbs?

yes other workflows that use .onnx models and insightface work as of an hr ago,
I can use the .pth landmark if need be to bypass the error but i will also do pip freeze to see what may be causing potential conflicts

**Comment by stoutimon** [comment]
I have this issue when running Pulid workflow

**Comment by Missing-Driver** [comment]
> I have this issue when running Pulid workflow

@stoutimon  I had the same issue after reinstalling ConfyUI into another drive. I copied the old files into the new models directory. On a fresh install of PulID, the necessary InsightFace models are supposed to be downloaded automatically, but only if they are not already present under the models directory. So what I did was to delete the antelopev2 directory (ComfyUI/ComfyUI/models/insightface/models/antelopev2), then I restarted Comfy, and then run a workflow with PulID in it, after running the workflow, antelopev2 was re-downloaded and my problem was solved.

**Comment by rohit7044** [comment]
Just delete "inswapper_128.onnx" inside the buffalo_i directory and keep "inswapper_128.onnx" inside insightface directory. You're welcome
